### PR TITLE
Ui cleanup

### PR DESCRIPTION
**Pull request by nithina**
@nishintn Please find the pull request containing following changes.
- added header and footer.
- changed router to ui-router.

Please review it,

http://10.9.12.143/html/dailyStatus/daily-status/

### CODE REVIEW

File: app/controllers/statusForm.js [hunk 1]
```diff
@@ -17,10 +16,10 @@ app.controller("StatusFormCtrl", ['$scope', '$http', 'generateNo', 'generateDays
             statusDate: daysArray[0],
             generateDates: daysArray,
             generateHrs: generateNo.paddednumbers(24),
-            generateMins: generateNo.paddednumbers(60),
-        }
+            generateMins: generateNo.paddednumbers(60)
```
Comment (mad-nishin) [inline app/controllers/statusForm.js]: Use variables instead of numerals directly in code so that its purpose is clear

**Comment by mad-nishin** [comment]
Comments :
1. Add a back action to go back to home
2. Style menu link 'Daily Status'
3. Align and CSS style contents
4. Avoid using br tags and instead CSS style.

**Comment by mad-nishin** [comment]
@nithina , added few feedback comments. Please implement them and re-submit the PR.

**Comment by nithina** [comment]
@nishintn Done style changes, please review.

**Comment by nithina** [comment]
@nishintn Done the changes based on feedback. Please review it.

http://10.9.12.143/html/dailyStatus/daily-status/

**Pull request merged**